src/styles/calendar.scss

```diff
@@ -0,0 +1,49 @@
+$square-size: 100px;
+$day-of-week: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+$width-calendar: 728px; //$square-size * 7 + 1(gap) * 8 + 10(padding) * 2
+
+.calendar {
+  width: $width-calendar;
+  border: 1px solid black;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  &__day {
+    background-color: #eee;
+    width: $square-size;
+    height: $square-size;
+    border: 1px solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-property: all;
```

Comment:
Let's not use `all`, write here styles to which you apply transition. For example `color`